### DISCUSSION TITLE
softcut feature: stop recording after the current loop+fade has completed

### DISCUSSION THREAD

**Discussion by schollz** [monome/norns#1635] [General]
This feature would allow softcut to record a single loop with the crossfaded post-buffer at the end to ensure seamless loops (idea from @catfact first spurned here: https://llllllll.co/t/oooooo/35828/269). It is something that can be done from the Lua side, but it is tricky as it involves timing out expanded loops and then slewing record levels and then changing the length of the loops. Its not impossible, but it would be nice to be able to do this with a single command without Lua tricks. I have an implementation that works as a single softcut command (`softcut.rec_once`): https://github.com/monome/softcut-lib/pull/59 

What are the thoughts about this feature? I'm not sure I ever closed the loop on it (no pun intended).

**Comment by schollz** [comment]
discussion continued here: https://github.com/monome/softcut-lib/pull/59#issuecomment-1365377294

**Comment by catfact** [comment]
this was done and merged yes? i'll close the discussion
